src/DemoHelperServiceProvider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Rapidez\DemoHelper;
+
+use Illuminate\Support\Facades\Route;
+use Illuminate\Support\Facades\View;
+use Illuminate\Support\ServiceProvider;
+use Rapidez\DemoHelper\Http\Middleware\SetDemoConfig;
+
+class DemoHelperServiceProvider extends ServiceProvider
+{
+    public function boot()
+    {
+        $this
+            ->bootMiddleware()
+            ->bootViews()
+            ->bootPublishables();
+    }
+
+    public function bootMiddleware(): self
+    {
+        Route::pushMiddlewareToGroup('web', SetDemoConfig::class);
+
+        return $this;
+    }
+
+    public function bootViews(): self
+    {
+        $this->loadViewsFrom(__DIR__.'/../resources/views', 'rapidez-demo-helper');
+
+        if (! $this->app->runningInConsole()) {
+            View::startPush('head', view('rapidez-demo-helper::top-bar'));
+        }
```

Comment:
This was kind of a bad idea previously, better to use a view composer. See: https://github.com/rapidez/statamic/pull/109/files

Reply:
Fixed 👍